CMakeLists.txt

```diff
@@ -52,5 +55,6 @@ target_link_libraries(ace ${SDL2_LIBRARY} ${SDL2_IMAGE_LIBRARIES}
                           ${ZLIB_LIBRARIES}
                           ${OPENAL_LIBRARY} ${ALURE_LIBRARY}
                           ${FREETYPE_LIBRARIES}
-                          ${CURL_LIBRARIES} 
-                          fmt::fmt)
\ No newline at end of file
+                          ${CURL_LIBRARIES}
+                          ${LIBDL_LIBRARY}
```

Comment:
Might be a good idea to wrap this in an if block, I'm not sure how this would be handled on Windows by CMake

Reply:
I tested it when `LIBDL_LIBRARY` isn't set, and it worked just as if I hadn't put it in the list.